### DISCUSSION TITLE
High DPI update

### DISCUSSION THREAD

**Discussion by MKleusberg** [sqlitebrowser/sqlitebrowser#2620] [Show and tell]
Hi everybody and sorry for the delay from my side. I have just started playing around with DB4S on that High DPI monitor. So far I have tried it at a resolution of 2560x1440 and at a resolution of 3840x2160. For the latter resolution I have tried a scaling of 100%, 125%, and 150%. In all configurations DB4S looks pretty normal. I expected it to look much worse to be honest :wink: 

![Screenshot_20210311_152542](https://user-images.githubusercontent.com/3153504/110802454-80016f80-827e-11eb-97df-f405ab1c0663.png)
![Screenshot_20210311_152644](https://user-images.githubusercontent.com/3153504/110802481-85f75080-827e-11eb-9511-dfdf73bcc644.png)
![Screenshot_20210311_152835](https://user-images.githubusercontent.com/3153504/110802490-87287d80-827e-11eb-9627-fadd511c2802.png)

This week I have still limited time but probably next week I'll do more experiments as trying this with a default configuration and on a Windows system. I'm fairly sure it's going to be a whole other experience on Windows.

**Comment by justinclift** [comment]
Excellent.  That's good news so far... :grinning:
